app/views/comments/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_with model: [@user, @post, @comment], url: user_post_comments_path do |f| %>
+  <div>
+    <%= f.label :text, required: true %>
+    <%= f.text_field :text, required: true %>
+  </div>
+  <div>
+    <%= f.submit "Submit" %>
+  </div>
```

Comment:
- [ ] [OPTIONAL] It would have been good if you created a button for adding comments to a post. Currently, users are typing the URL to find this form.